plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -502,4 +508,9 @@ public static boolean isSortedWritingEnabled(ConnectorSession session)
     {
         return session.getProperty(SORTED_WRITING_ENABLED, Boolean.class);
     }
+
+    public static boolean isFilterOnPartitionTable(ConnectorSession session)
```

Comment:
Can you please rename this to be in line with the config name?
e.g. `isQueryPartitionFilterRequired()`